db/migrate_cards/data/renaming_initiative.json

```diff
@@ -0,0 +1,31 @@
+{
+  "url": null,
+  "timestamp": "2016-02-17 18:07:36 +0000",
+  "card": {
+    "name": "export",
+    "type": "Pointer",
+    "content": "[[project+open]]\n[[project+Closed]]\n[[project+*self+*structure]]\n[[nav bar menu]]",
+    "value": [
+      {
+        "name": "Project+Open",
+        "type": "Search",
+        "content": "{ \r\n  \"type\":\"Project\",\r\n  \"right_plus\":[\"status\",{\"refer_to\":\"_right\"}]\r\n}"
+      },
```

Comment:
It will rename the card name but not the content. Some of them have `[[Initiative]]` which are not updated automatically.